nodeadm/Makefile

```
@@ -5,6 +5,10 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+# Version from git tags
+VERSION ?= $(shell git describe --tags --abbrev=8 --always)
```

Unfortunately, this won't work with our current setup. The `nodeadm` build happens in a container as a part of the AMI build, and there is no `.git` directory in that workspace. Additionally, the `git` tag for an release does not actually exist when the AMI build happens. We could wire something up, but it would have to happen in our internal release tooling.

We could do this as a pre-req without changing behavior, and then set the version in a future release:
```suggestion
VERSION ?= 0.0.0
```

---

Umm... interesting, sounds like we need somethings like sed/awk/...